base_projects/Drivetrain/src/main/cpp/Robot.cpp

```diff
@@ -0,0 +1,367 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#include "Robot.h"
+
+#include <rev/CANSparkMax.h>
+
+#include <cstddef>
+#include <iostream>
+#include <sstream>
+
+#include <frc/ADXRS450_Gyro.h>
+#include <frc/AnalogGyro.h>
+#include <frc/Filesystem.h>
+#include <frc/RobotController.h>
+#include <frc/Spark.h>
+#include <frc/TimedRobot.h>
+#include <frc/romi/RomiGyro.h>
+#include <frc/smartdashboard/SmartDashboard.h>
+#include <units/angular_velocity.h>
+#include <wpi/FileSystem.h>
+#include <wpi/SmallString.h>
+#include <wpi/StringMap.h>
+
+// #include "AHRS.h"
+#include "rev/CANEncoder.h"
+
+Robot::Robot() : frc::TimedRobot(5_ms) {
```

Comment:
To improve scheduling jitter, we should also set the HAL Notifier thread and the TimedRobot thread to RT, like this: https://github.com/frc3512/Robot-2020/blob/main/src/main/cpp/Robot.cpp#L87-L93. 3512 uses a priority of 15.

Reply:
Should this be something we might abstract into the Logging API?

Reply:
Abstract in what way?

Reply:
Is [this](https://github.com/Piphi5/sysid/blob/generation/integration_test_project/src/main/cpp/logging/SysIdLogger.cpp#L41-#L48) a good spot to put it?